packages/app-mobile/components/NoteEditor/ImageEditor/ImageEditor.tsx

```diff
@@ -124,10 +124,10 @@ const ImageEditor = (props: Props) => {
 			onRequestCloseEditor(true);
 			return true;
 		};
-		BackHandler.addEventListener('hardwareBackPress', hardwareBackPressListener);
+		const handle = BackHandler.addEventListener('hardwareBackPress', hardwareBackPressListener);
 
 		return () => {
-			BackHandler.removeEventListener('hardwareBackPress', hardwareBackPressListener);
+			handle.remove();
```

Comment:
React Native deprecated and removed `BackHandler.removeEventListener` in React Native 0.77.